render/expected/expected.go

```diff
@@ -17,18 +17,21 @@ var (
 		ID:                unknownPseudoNode1ID,
 		LabelMajor:        "10.10.10.10",
 		Pseudo:            true,
+		NodeMetadata:      report.MakeNodeMetadata(map[string]string{}),
```

Comment:
Well, sure, the tests will pass, but the output of trimNodeMetadata will leave the report in an invalid state. A instance of NodeMetadata with a nil Metadata member is invalid.